lib/commands/publish.js

```diff
@@ -21,7 +24,11 @@ exports.execute = function (config) {
   //
 
   try {
-    checkUpdatedPackages();
+    changedPackages = checkUpdatedPackages(config);
+    console.log('Packages to be updated');
+    console.log(changedPackages.map(function(pkg) {
+      return "- " + pkg;
+    }).join('\n'));
```

Comment:
Yeah I forgot to run lint after I pasted your snippet - I just added travis which will run lint for now